src/components/display/components/LinkComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { useNavigation } from '@react-navigation/core'
+import { LinkArgs } from 'display-api'
+import { Linking } from 'react-native'
+import { TouchableRipple } from 'react-native-paper'
+import { redirectTo } from '../../../utils/helper'
+import { TextComponent } from './TextComponent'
+
+export const LinkComponent = ({ text, external, app_path, url }: LinkArgs) => {
+	const navigation = useNavigation()
+
+	const onPress = () => {
+		if (app_path) {
+			redirectTo(app_path, navigation)
+		} else if (external && url && Linking.canOpenURL(url)) {
```

Comment:
`external` is more for the web. It's for dictating when to open the link in a new tab, so for mobile it should always open it externally.